src/lib/types/model.ts

```diff
@@ -514,6 +514,7 @@ export enum RoleName {
     VIEWER = 'Viewer',
     OWNER = 'Owner',
     MEMBER = 'Member',
+    READER = 'Reader',
```

Comment:
If you relaxed the constraint, can't you use "Viewer"? I'm ok with reader as well, just asking

Reply:
We can, but it felt like we should use a unique name if possible just to prevent confusion between predefined root and project roles.